src/pytest_servers/factory.py

```diff
@@ -111,3 +125,20 @@ def memory_temp_path(self, **kwargs) -> UPath:
             f"memory://{random_string()}",
             **kwargs,
         )
+
+    def gcs_temp_path(
+        self, endpoint_url: Optional[str] = None, **kwargs
+    ) -> UPath:
+        """Creates a new gcs bucket and returns an UPath instance.
+
+        `endpoint_url` can be used to use custom servers
+        (e.g. fake-s3-server).
```

Comment:
Does it work with gcs? O_o

Reply:
Meant to type `fake-gcs-server` :slightly_smiling_face: